### DISCUSSION TITLE
show carrier please (like mobile ookla) and also provide daily average speeds

### DISCUSSION THREAD

**Discussion by mrpid12** [alexjustesen/speedtest-tracker#1551] [Ideas]
Hi, I have dual WAN SD-WAN with two mobile providers - it would be great to show which carrier was used in the speedtest, like you can see when you use oolka on a mobile phone. that way I could be sure which network the load balancer sent a speedtest through. currently I do two concurrent speedtests from seprate machnes inside my network, and get two different results (as I would expect, the ladbalancer sends one down each carrier). I just cant be sure which result was which.  ALso, daily average speeds would be great to see, thanks

**Comment by svenvg93** [comment]
Hi @mrpid12, 

In the Results table you can enable to show the ISP used during the speedtest. 
The Daily average is already reqeuisted here https://github.com/alexjustesen/speedtest-tracker/issues/1541. Keep an eye on for the progress :)

**Comment by mrpid12** [comment]
Many thanks for the pointer, much appreciated!

**Reply by svenvg93** [reply]
your welcome!
